tests/wallet/nft_wallet/test_nft_wallet.py

```diff
@@ -253,7 +253,7 @@ async def test_nft_wallet_creation_and_transfer(two_wallet_nodes: Any, trusted:
         assert False
     await full_node_api.reorg_from_index_to_new_index(ReorgProtocol(uint32(height - 1), uint32(height + 1), ph1, None))
     await time_out_assert(30, get_nft_count, 0, nft_wallet_0)
-    await time_out_assert(30, get_wallet_number, 1, wallet_node_0.wallet_state_manager)
+    await time_out_assert(30, get_wallet_number, 2, wallet_node_0.wallet_state_manager)
```

Comment:
so we create a new wallet here or why do we have 2 wallets now?

Reply:
The original XCH wallet and DID wallet